packages/trader/src/Stores/client-store.js

```diff
@@ -976,5 +981,10 @@ export default class ClientStore extends BaseStore {
         }
         return [];
     }
+
+    @action.bound
+    getRiskAssessment() {
```

Comment:
can the risk assessment be a computed property on the client store instead?

Reply:
@oskar-binary ideally yes we could, but the `computed` value always returns `undefined` on the first render :(